playbooks/robusta_playbooks/job_restart_on_oomkilled.py

```diff
@@ -1,5 +1,26 @@
-from robusta.api import *
+from robusta.api import (
+    action,
+    ActionParams,
+    Optional,
+    JobEvent,
+    Finding,
+    Container,
+    FindingSource,
+    logging,
+    FindingType,
+    PodContainer,
+    MarkdownBlock,
+    ResourceRequirements,
+    RobustaJob,
+    ObjectMeta,
+    JobSpec,
+    PodSpec,
+    PodTemplateSpec,
+)
 from ...src.robusta.core.playbooks.job_utils import get_job_latest_pod
+from bitmath import KiB, GiB, MiB
```

Comment:
I think bitmath module does not come out of the box. It should be added to the pyproject.toml dependency file with a specific version (e.g bitmath="^1.3.3")

Reply:
added.